api-gateway/src/main/kotlin/com/saveourtool/save/gateway/utils/StoringServerAuthenticationSuccessHandler.kt

```diff
@@ -25,11 +23,7 @@ class StoringServerAuthenticationSuccessHandler(
 ) : ServerAuthenticationSuccessHandler {
     private val logger = LoggerFactory.getLogger(javaClass)
     private val objectMapper = ObjectMapper()
-        // This is necessary to remove the originalLogins from the User,
-        // otherwise we will get the kotlin.EmptyList class in Json and will not be able to serialize it.
-        .configure(SerializationFeature.WRITE_EMPTY_JSON_ARRAYS, false)
         .findAndRegisterModules()
-        .registerModule(CoreJackson2Module())
         .registerModule(OAuth2ClientJackson2Module())
```

Comment:
This is not needed either, moreover, if you look in their source code, they both call `SecurityJackson2Modules.enableDefaultTyping(context.getOwner());` which must be the cause of weird array serialization

Reply:
maybe we don't need to initialize objectMapper and take it from spring context?

Reply:
Fixed